tests/services/test_service.py

```diff
@@ -136,3 +136,22 @@ def test_missing_or_invalid_type(lang_data2, service, identity):
         lang_data2['props']['newkey'] = v
         pytest.raises(
             ValidationError, service.create, identity, lang_data2)
+
+
+def test_read_all_no_cache(lang_type, lic_type, lang_data_many, service, identity):
+    """ read_all method should return all languages created in this scope. """
+
+    items = service.read_all(identity, fields=[], type='languages', cache=False)
+    assert set(lang_data_many).issubset(set([i.id for i in items._results])) 
+    #assert len(items._results) == len(lang_data_many) #TODO: Fix it to actual number with scope='module'
+
+
+def test_read_many(lang_type, lic_type, lang_data_many, service, identity):
+    """ read_many method should return all requested languages. """
+
+    items = service.read_many(identity, type='languages', ids=['fr','tr','es'], fields=[])
+    item_ids = [i.id for i in items._results]
+    for id_ in ['fr','tr','es']:
+        assert id_ in item_ids
```

Comment:
faster with sets?
```python
assert {'fr', 'tr', 'es'} == set(item_ids)
```

Shouldn't be a subset, since they should be the *only* ones returned.

Reply:
elegant, thought of it after push